.github/workflows/ci.yml

```diff
@@ -197,6 +206,36 @@ jobs:
           path: ./skiko/build/reports/tests
           retention-days: 5
 
+      - uses: actions/upload-artifact@v4
+        name: 'Save linuxArm64 test binary as artifact'
+        with:
+          name: test-binary-linuxArm64
+          path: ./skiko/build/bin/linuxArm64
+          if-no-files-found: error
+          retention-days: 5
+
+  linux-arm:
+    needs: linux
+    runs-on: ubuntu-22.04-arm
+    steps:
+      - uses: actions/checkout@v3
+        name: 'Check out code'
+      - uses: actions/download-artifact@v4
+        with:
+          name: test-binary-linuxArm64
+          path: ./skiko/build/bin/linuxArm64
+      - shell: bash
```

Comment:
You don't need to specify the shell here. I do it on Mosaic because my job is a matrix that also runs on Windows where the default is power shell.